CMakeLists.txt

```diff
@@ -555,8 +537,8 @@ if(SILO_ENABLE_HDF5 AND HDF5_FOUND)
     endif()
 
     if(SILO_ENABLE_FPZIP)
+        set(HAVE_FPZIP 1)
```

Comment:
Where else is this used? Does it have to be an integer value? bool (true) might be better.
Simliar question for HAVE_HZIP below.

Reply:
Through `config.h.cmake.in` --> `config.h`

Reply:
Thank you for the clarification.